samples/msal-node-samples/ElectronSystemBrowserTestApp/src/config/customConfig.json

```diff
@@ -7,7 +7,7 @@
     "resourceApi":
     {
         "endpoint": "Enter_the_Graph_Endpoint_Here",
```

Comment:
> "endpoint": "Enter_the_Graph_Endpoint_Here"

I always get confused if this should be `https://graph.microsoft.com`, `graph.microsoft.com/v1.0` or `graph.microsoft.com/v1.0/me` -let's hardcode this (if anyone wants to use the sample with national authorities they can figure this out)

Reply:
Likewise, let's hardcode `Enter_the_Cloud_Instance_Id_Here`

Reply:
Done